### PR TITLE
PWX-28289 Add CSV for 1.10.2

### PR DESCRIPTION
**Pull request by haibinxie**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: deploy/olm-catalog/portworx/1.10.2/portworx-certified.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+                  - labelSelector:
+                      matchExpressions:
+                      - key: name
+                        operator: In
+                        values:
+                        - portworx-operator
+                    topologyKey: kubernetes.io/hostname
+              containers:
+              - name: portworx-operator
+                image: registry.connect.redhat.com/portworx/openstorage-operator@sha256:b25747be8db143924e651f25e86d95b7013da5be0de81796c5d70bbf9ecfb511
```
Comment (pure-jliao) [inline deploy/olm-catalog/portworx/1.10.2/portworx-certified.clusterserviceversion.yaml:218]: change this as well

**Comment by pure-jliao** [comment]
found a previous PR, 2.0 csv needs update as well https://github.com/libopenstorage/operator/pull/759/files#diff-3744506451d957f37c75135e8a346fbdb60247513566b8ac81e4f124e76dcf4b

**Review by pure-jliao — APPROVED** [review]
2.0 csv needs change, other than that looks good.

**Comment by haibinxie** [comment]
> 2.0 csv needs change, other than that looks good.

I intended to not change it, as 2.0 has been there for long time and never used, 1.11.0 is next release.

**Comment by haibinxie** [comment]
> 2.0 csv needs change, other than that looks good.

actually let me send a PR to delete 2.0

**Comment by piyush-nimbalkar** [comment]
> > 2.0 csv needs change, other than that looks good.
> 
> actually let me send a PR to delete 2.0

Remember that we used to use the 2.0 CSV for master. It was useful when someone wanted to test an upgrade to master. Especially for automation.

The issue right now is that out master is not clean and we never test or release anything from master. Once we address that we should keep a CSV for master which should be different from the latest release branch version.

**Pull request merged**